test/models/real-fevr-marketplace.spec.ts

```diff
@@ -0,0 +1,78 @@
+import {defaultWeb3Connection, erc20Deployer, revertChain} from '../utils/';
+import {expect} from 'chai';
+import {RealFevrOpener} from '@models/real-fevr-opener';
+import {RealFevrMarketplace} from '@models/real-fevr-marketplace';
+import {toSmartContractDecimals} from '@utils/numbers';
+
+describe('Marketplace RealFevr', async () => {
+  let marketplaceContract: RealFevrMarketplace;
+  let tokenAddress!: string;
+  let openerAddress!: string;
+  let contractAddress!: string;
+  let accountAddress!: string;
+
+  const purchaseTokenCap = toSmartContractDecimals(1000000) as number;
+  const web3Connection = defaultWeb3Connection();
+
+  before(async () => {
+    await web3Connection.start();
+    await revertChain(web3Connection.Web3);
+    accountAddress = web3Connection.Account.address;
+  });
+
+  it(`Deploys the contract`, async () => {
```

Comment:
be specific, the test refers a market place but this it relies on 3 contracts to be deployed ;)

Reply:
done, indeed I copy-pasted and forgot to account for the added logic 👍 split it all up, should be much clearer now